### PR TITLE
Survey project. Vladimir Bykov

### PR DESCRIPTION
**Pull request by vladjnbykov**
Link to my deployed project: https://awesome-beaver-ae1ba7.netlify.app/

My reflections on how this week's project turned out: Very tough. Information flow between components is the major issue between 

Things I'd like to have clarified or explained in more detail: Connections between components

### CODE REVIEW

File: code/src/components/Summary.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      <h1 className="underscore">Summary:</h1>
+      <h2>{fname} {lname}</h2>
+      <h3>Gender: {gender}</h3>
+      <h3>In the past 6 months I {anamnes}</h3>
+      <h3>Currently I feel {checkboxGroup.join(', ')}</h3>
+    </div>
+  )
+}
+
+export default Summary
```
Comment (Ulve17) [inline code/src/components/Summary.js:17]: 👍

File: code/src/components/FormQuestions.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        className="box"
+        onChange={onAnamnesChange}
+        value={anamnes}>
+        <option value="">In the past 6 months did you have:</option>
+        <option value="had COVID-19">COVID-19 infection</option>
+        <option value="had antibodies against coronavirus">Antibodies against coronavirus</option>
+        <option value="had positive swab test for coronavirus">Positive swab test for coronavirus</option>
+        <option value="had vaccination against coronavirus">Vaccination against COVID-19</option>
+        <option value="am not aware of been infected by COVID-19">I am not aware of been infected by COVID-19</option>
+      </select>
```
Comment (Ulve17) [inline code/src/components/FormQuestions.js:23]: Nice code!! 👍

File: code/src/components/Checkboxes.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      </div>
+
+      <div>
+        <label htmlFor="headache">Headache</label>
+        <input
+          id="headache"
+          type="checkbox"
+          checked={checkboxGroup.includes('headache')}
+          onChange={() => onCheckboxToggle('headache')} />
+      </div>
```
Comment (Ulve17) [inline code/src/components/Checkboxes.js:18]: good job !

File: code/src/components/Form.js [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+const genderGroups = ['male', 'female', 'other']
+
+const Form = () => {
+  const [fname, setFname] = useState('')
+  const [lname, setLname] = useState('')
+  const [gender, setGender] = useState('')
+  const [anamnes, setAnamnes] = useState('')
+  const [IsSubmited, setIsSubmited] = useState(false)
+  const [checkboxGroup, setCheckboxGroup] = useState([])
```
Comment (Ulve17) [inline code/src/components/Form.js:17]: 👍

File: code/src/index.css [hunk 5]
```diff
@@ -5,9 +5,83 @@ body {
...
+input[type=submit] {
+  background-color: rgb(60, 255, 0);
+  border: 2px solid navy;
+  font-size: 15px;
+  font-weight: bold;
+}
+
+.underscore {
+  text-decoration: underline;
+}
```
Comment (Ulve17) [inline code/src/index.css:87]: Nice styling 😄

**Review by Ulve17 — APPROVED** [review]
Hi vlad! Hope you are doing well.
Great job with the project this week! 😸 Hopefully i didn't miss anything. 😨 
Take care 😄 
Ulysses

**Review by KarolCislowski — APPROVED** [review]
Great job! 

For the next step, it would be nice to try separate questions to multiple pages.

**Pull request closed without merging**